### ISSUE TITLE
I have one question, does your latest version v7.2+ work with the latest version of react native 0.67???

### ISSUE THREAD

**Issue by Islam312** [microsoft/react-native-code-push#2237]

**Comment by FadiAboMsalam** [comment]
@Islam312  i am using same versions and facing same issue ,  did you manage to make it work?

**Comment by Islam312** [comment]
> @Islam312 i am using same versions and facing same issue , did you manage to make it work?

Unfortunately not working yet

**Comment by FadiAboMsalam** [comment]
Did anyone test this on RN 0.68.0 ?

**Comment by fragilehm** [comment]
Hey folk, my app is in `0.67.1` and the codepush seems to work, I am planning to update to `0.67.4` and that is not working?

**Comment by FadiAboMsalam** [comment]
@fragilehm  yes on 0.67.4 it doesn't work yet

**Comment by pnthach95** [comment]
I'm using it in production, work well on Android and iOS

```
"react-native": "0.67.4",
"react-native-code-push": "^7.0.4",
```

**Comment by gcourtemanche** [comment]
@fragilehm thank you 💯 

I'm also on `0.67.4` and iOS was working but not android.
The linked comment did the trick!

**Comment by FadiAboMsalam** [comment]
On ios it works but on android after code push installed the app crash and rolls back always am not sure why the error on log cat 

`Fatal Exception: com.facebook.react.common.c
TypeError: Cannot read property 'split' of undefined, js engine: hermes, stack: anonymous@738:218 h@2:1585 d@2:1028 o@2:496 anonymous@731:127 h@2:1585 d@2:1028 o@2:496 anonymous@730:117 h@2:1585 d@2:1028 o@2:496 anonymous@677:150 h@2:1585 d@2:1028 o@2:496 anonymous@6:67 h@2:1585 d@2:958 o@2:496 global@5163:3`

**Comment by FadiAboMsalam** [comment]
I believe the issue or at least my issue related directly to this [issue](https://github.com/microsoft/react-native-code-push/issues/2258)

**Comment by hth-developers** [comment]
I'm also on 0.67.4 and iOS was working but not android.

**Comment by orzhtml** [comment]
ios Flash back

   OS: macOS 12.4
    CPU: (8) arm64 Apple M2
    Memory: 111.84 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 18.5.0 - ~/.nvm/versions/node/v18.5.0/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v18.5.0/bin/yarn
    npm: 8.12.1 - ~/.nvm/versions/node/v18.5.0/bin/npm
    Watchman: 2022.07.04.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK: Not Found
  IDEs:
    Android Studio: EAP AI-221.5921.22.2211.8786657 AI-221.5921.22.2211.8786657
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.15 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.0.0 => 18.0.0 
    react-native: 0.69.3 => 0.69.3

**Comment by Chronostyle** [comment]
@fragilehm thx this also fixed my issue #2350 in the code push 7.0.5 and rn 0.70.1
